src/maidsafe/vault/mpid_manager/dispatcher.cc.orig

```diff
@@ -0,0 +1,142 @@
+/*  Copyright 2013 MaidSafe.net limited
```

Comment:
I'll merge anyway since this is such a large PR.  Please remove it in the next Vault task.